wesnoth-dw.po

```diff
@@ -2471,7 +2411,7 @@ msgid ""
 "long while. I bet I can wake him up, though. Watch THIS!"
 msgstr ""
 "No, říká si teď Mal-Govon, ale je to on. Nazval tuto zem KRÁLOVSTVÍM "
-"AGNOVON. Stal se jeho prvním králem. A také jeho posledním a také všemi mezi "
+"Agnovon. Stal se jeho prvním králem. A také jeho posledním a také všemi mezi "
```

Comment:
chtěl jsem to opravit podle vás v master, ale všiml jsem si kontextu - Agnovon vypadá jako další jméno toho licha

takže si dovoluji navrhnout buď Agnovonovo KRÁLOVSTVÍ, nebo
```suggestion
"Agnovona. Stal se jeho prvním králem. A také jeho posledním a také všemi mezi "
```